client/a8c-for-agencies/sections/partner-directory/components/hooks/use-form-selectors.ts

```diff
@@ -58,11 +70,25 @@ export function useFormSelectors() {
 		pressable: 'Pressable.com',
 	};
 
+	const oldIndustries: Record< string, string > = {
+		agricultural_services: 'agriculture_and_farming',
+		business_services: 'legal_and_professional_services',
+		clothing_shops: 'fashion_and_beauty',
+		contracted_services: 'legal_and_professional_services',
+		government_services: 'government_and_public_services',
+		miscellaneous_shops: 'e_commerce_and_retail',
+		professional_services_and_membership_organisations: 'legal_and_professional_services',
+		retail_outlet_services: 'e_commerce_and_retail',
+		transportation_services: 'automotive_and_transportation',
+		utility_services: 'energy_and_utilities',
+	};
+
 	return {
 		availableServices,
 		availableLanguages,
 		availableProducts,
 		availableDirectories,
-		availableIndustries,
+		availableIndustries: Object.fromEntries( Object.entries( availableIndustries ).sort() ),
```

Comment:
Is there a specific reason why we are performing the sort operation here? Can't we simply define the items in the order we'd like?

Reply:
I thought it's better to sort it here as there is a chance that we might miss updating it alphabetically. I have now made the changes and added a comment.